components/sheet/src/sheet.tsx

```diff
@@ -28,7 +28,7 @@ const SheetOverlay = forwardRef<
 SheetOverlay.displayName = SheetPrimitive.Overlay.displayName;
 
 const SheetVariants = cva(
-	"fixed z-50 flex flex-col bg-background shadow-lg transition ease-in-out data-state-open:animate-in data-state-closed:animate-out data-state-closed:duration-100 data-state-open:duration-100",
+	"fixed z-50 flex flex-col bg-base shadow-lg transition ease-in-out data-state-open:animate-in data-state-closed:animate-out data-state-closed:duration-300 data-state-open:duration-500",
```

Comment:
```suggestion
	"fixed z-50 flex flex-col bg-base shadow-lg transition ease-in-out data-state-open:animate-in data-state-closed:animate-out data-state-closed:duration-100 data-state-open:duration-100",
```

My merge fail. We want these to be quicker now.